data/remnant/remnant.txt

```diff
@@ -507,7 +634,10 @@ phrase "friendly remnant"
 		" "
 	word
 		"buying"
+		"equipping"
 		"getting"
+		"installing"
+		"outfitting"
```

Comment:
I don't think this works here. I don't outfit some Inhibitor Cannons, I outfit my ship with some Inhibitor Cannons.
```suggestion
		"outfitting your ship with"
```